.github/workflows/test.yml

```
@@ -11,8 +11,28 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        platform: [ubuntu-latest]
+        platform:
```

How do you think the explicit expression like following:

```suggestion
        include:
          - platform: ubuntu-latest
             python-version: "3.9"
          - platform: ubuntu-latest
             python-version: "3.10"
             ...
           - platform: macos-latest
             python-version: "3.13"
```

---

I respected the original writing style, but this one seems better!